### DISCUSSION TITLE
Request to present an app at the next Open Visualization Meeting

### DISCUSSION THREAD

**Discussion by chjch** [visgl/deck.gl#8521] [Show and tell]
We developed the Resilient Cedar Key (FL) app https://resilientcedarkey.web.app/ using deck.gl. The app has been very well received by the local community; and it has been very instrumental in communicating future flood risks under the sea-level rise context. I hope to share our story of using deck.gl with you next Thursday (2/29).
![CedarKey_Workshop_Users](https://github.com/visgl/deck.gl/assets/12037254/586a5030-025f-469c-b3a1-1d0b092a2331)

**Comment by ibgreen** [comment]
@chjch We'd love to have you present your app at our bi-weekly meeting. Based on our off-line discussion sounds like Thursday March 14 works best for you. @chrisgervang 

I would encourage you to open a PR to add your application to our showcases page. There are many closed PRs that you can use as reference for instance https://github.com/visgl/deck.gl/pull/8190/files. @Pessimistress
